platform-configs/include/msi-z690-common.robot

```diff
@@ -40,6 +40,9 @@ ${WIFI_CARD_UBUNTU}=                            ${TBD}
 ${USB_MODEL}=                                   Kingston
 ${USB_DEVICE}=                                  Multifunction Composite Gadget
 
+${INTEL_HYBRID_ARCH_SUPPORT}=                   ${FALSE}
```

Comment:
If it relies on `rest to defaults`, we may give up on running these tests automatically right now as well: https://github.com/Dasharo/open-source-firmware-validation/pull/530/files#r1806136564

Or at least leave it until we have all of the "old" automation working in place.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/issues/540